.github/workflows/daily_common.yml

```diff
@@ -89,6 +89,21 @@ jobs:
         run: |
           nimble install -y --depsOnly
 
+      - name: Compile nimbus using nim-libp2p
+        if: ${{ matrix.platform.os == 'linux' && matrix.cpu == 'amd64' && matrix.nim.ref == 'version-2-2'}}
+        run: |
+          nim --version
+          nimble --version
+
+          export NIMFLAGS="${NIMFLAGS} --mm:${{ matrix.nim.memory_management }}"
+
+          git clone --branch unstable --single-branch https://github.com/status-im/nimbus-eth2.git
+          cd nimbus-eth2
+          git submodule set-branch --branch ${{ github.sha }} vendor/nim-libp2p
+
+          make -j"$(nproc)"
```

Comment:
Isn't it necessary to execute `make update`?
nimbus-build-system README says:
> Instruct your users to run make update after cloning your project, after a git pull or after switching branches.

Reply:
Weird, [the build guide doesn't mention `update`](https://nimbus.guide/build.html)